### PR TITLE
[Imagery] Update metadata for images

### PR DESCRIPTION
**Pull request by VWoeltjen**
Opening for status and todo-tracking. Already completed by @larkin:

* Convert example imagery to single-file plugin, with "image" hint
* Update image view policy to use new-style domain objects
* Update image view controller to use new-style domain objects, new telemetry API

Still to-do:

- [x] Reconcile field names: Both imageUrl or image in ImageryViewController
- [x] Finish specs
- [x] Documentation
- [x] Satisfy code style checks
- [x] Smoke test again

### CODE REVIEW

**Comment by VWoeltjen** [comment]
### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

File: platform/features/imagery/src/controllers/ImageryController.js [hunk 1]
```diff
@@ -141,8 +122,8 @@ define(
         ImageryController.prototype.paused = function (state) {
             if (arguments.length > 0 && state !== this.isPaused) {
                 this.isPaused = state;
-                // Switch to latest image
-                this.updateValues();
+                this.updateValues(this.nextValue);
```
Comment (larkin) [inline platform/features/imagery/src/controllers/ImageryController.js]: Ah, if you pause and unpause without a new value coming in, this.nextValue is undefined and it clears the displayed image.  Easy to fix:

```javascript
if (this.nextValue) { 
    this.updateValues(this.nextValue); 
    delete this.nextValue; 
}
```

**Comment by larkin** [comment]
@VWoeltjen One small bug to fix and otherwise this looks good.  I'll leave a review checklist and you can merge when it's fixed.

# Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? Y
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Comment by VWoeltjen** [comment]
Updated with requested changes. Additionally changed `nextValue` to `nextDatum` to clarify what is stored there (it threw me off briefly when reasoning about this change)

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Y
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

**Pull request merged**